app/utils/markdown/index.ts

```diff
@@ -363,3 +365,12 @@ export function convertSearchTermToRegex(term: string): SearchPattern {
         term,
     };
 }
+
+export const removeImageProxyForKey = (key: string) => {
+    const parseKey = parseUrl(key, true);
+    if (parseKey.query.url) {
```

Comment:
are we targeting a specific image proxy? if so then this is ok, if not we should probably go through the different query parameters and check.
E.g.: Imgproxy uses sourceurl, which would fail to retrieve it: https://docs.imgproxy.net/usage/processing#source-url